projects/storefrontlib/src/cms-components/checkout/config/checkout-config.ts

```diff
@@ -1,11 +1,21 @@
 import { CheckoutStep } from '../model/checkout-step.model';
 
+export enum DeliveryModePreferences {
+  FREE = 'FREE',
+  LEAST_EXPENSIVE = 'LEAST_EXPENSIVE', // but not free
+  MOST_EXPENSIVE = 'MOST_EXPENSIVE',
+}
+
 export abstract class CheckoutConfig {
   checkout?: {
     steps: Array<CheckoutStep>;
     /**
      * Allow for express checkout when default shipping method and payment method are available.
      */
     express?: boolean;
+    /**
+     * Pre-selected delivery mode. Allowed value is an array of DeliveryModePreferences or delivery codes.
```

Comment:
```suggestion
     * Default delivery mode for express checkout. Set preferences in order with general preferences (eg. DeliveryModePreferences.LEAST_EXPENSIVE) or specific delivery codes.
```

Reply:
That will be not only for express checkout.